addons/website_event/controllers/main.py

```diff
@@ -203,8 +204,17 @@ def event_register(self, event, **post):
     def _prepare_event_register_values(self, event, **post):
         """Return the require values to render the template."""
         urls = lazy(event._get_event_resource_urls)
+        available_slots_by_date = event.sudo().event_slot_ids.filtered(
+            lambda s: s.start_datetime > datetime.now()
+            and (not s.event_id.seats_max or s.seats_available > 0)
+        ).grouped('date')
         return {
             'event': event,
+            'slots': available_slots_by_date,
```

Comment:
Would do "slots_by_date" then, otherwise you kind of expect a browserecord of slots.